avro-fastserde/src/main/java/com/linkedin/avro/fastserde/FastSerializer.java

```diff
@@ -0,0 +1,10 @@
+package com.linkedin.avro.fastserde;
+
+import java.io.IOException;
+import org.apache.avro.io.Encoder;
+
+
+public interface FastSerializer<Type> {
```

Comment:
Same as above: recommend calling the generic type `T` or `TYPE`.

Reply:
Will change it to `T`.